sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 5.1.0-beta.4 (Unreleased)
 
+### Breaking changes
+- Renamed `RecognizeHealthcareEntitiesResultCollection` to `RecognizeHealthcareEntitiesResultCollection`.
+- Renamed `DocumentHealthcareResult` to `AnalyzeHealthcareEntitiesResult`.
+- Removed `StartHealthcare` and `StartHealthcareAsync` methods.
+- Renamed `StartHealthcareBatch` and `StartHealthcareBatchAsync` to `StartAnalyzeHealthcareEntities` and `StartAnalyzeHealthcareEntitiesAsync` respectively.
+- Renamed `HealthcareOptions` to `AnalyzeHealthcareEntitiesOptions`.
```

Comment:
it will be good to also add that the cancel method is now inside of the operation instead of the client